<launch>

 <!--Machine Definition-->

	<machine name="small"              
	address="10.42.0.232"                       
	user="ubuntu"                             
	ssh-port = "22"                            
	env-loader="~/env.sh"                       
	timeout="20" />         

     <machine name="master"
     address="10.42.0.1"
     user="ubuntu"
     ssh-port = "22"
     env-loader="~/env.sh"
     timeout="20" />

<!--small_machine-->

 <node pkg="main_loop" type="path_planning_" name="path_planning_" output="log" respawn="true" ns="small" machine="small">
 </node>

 <node pkg="main_loop" type="main_demo_v5" name="main_demo_v5" output="screen" respawn="true" ns="small" machine="small">
 </node>
 <node pkg="main_loop" type="agent_new_5" name="agent_new_5" respawn="true" ns="small" machine="small" >
 <!--<remap from="update_status" to="/master/pub_status" />-->
 </node>
 <node pkg="main_loop" type="main_demo_4.py" name="main_demo_5" respawn="true" ns="small" machine="small" >
 </node>
 <!--<node pkg="eurobot_gui" type="eurobot_gui.py" name="eurobot_gui" output="screen" respawn="true" ns="small" machine="small" >
 </node>-->
 
 

 
    
<!--master_machine-->

<node pkg="main_loop" type="path_planning_" name="path_planning_" output="log" respawn="true" ns="master" machine="master">
 </node>

 <node pkg="main_loop" type="main_demo_v4" name="main_demo_v4" output="screen" respawn="true" ns="master" machine="master">
 </node>
 <node pkg="main_loop" type="agent_new_4" name="agent_new_4" respawn="true" ns="master" machine="master" >
 </node>
 <node pkg="main_loop" type="main_demo_4.py" name="main_demo_4" respawn="true" ns="master" machine="master" >
 </node>
 <node pkg="eurobot_gui" type="eurobot_gui.py" name="eurobot_gui" output="screen" respawn="true" ns="master" machine="master" >
 </node>
 
 
 
</launch>
